### ISSUE TITLE
Feature request: Remove CompatTelRunner.exe automatically

### ISSUE THREAD

**Issue by nslay** [Sycnex/Windows10Debloater#426]
I'm sure CompatTelRunner is great/OK on SSDs, but it is not an exaggeration to say that CompatTelRunner completely dominates disk bandwidth on mechanical disks. I have a beefy Haswell-E system (yeah, I know it's a little dated) that is brought to its knees by CompatTelRunner. It would be really cool if Windows10Debloater could just automatically remove CompatTelRunner.

Upon noticing significant performance degradation and solid read/write HDD LED, I currently either manually remove CompatTelRunner or I use Process Explorer to suspend it indefinitely. It's almost always the cause of significant performance degradation for me (again, it may sound like an exaggeration, I'm not exaggerating).

By the way, thanks for your work on this amazing project!

**Comment by God-damnit-all** [comment]
> I'm sure CompatTelRunner is great/OK on SSDs, but it is not an exaggeration to say that CompatTelRunner completely dominates disk bandwidth on mechanical disks. I have a beefy Haswell-E system (yeah, I know it's a little dated) that is brought to its knees by CompatTelRunner. It would be really cool if Windows10Debloater could just automatically remove CompatTelRunner.
> 
> Upon noticing significant performance degradation and solid read/write HDD LED, I currently either manually remove CompatTelRunner or I use Process Explorer to suspend it indefinitely. It's almost always the cause of significant performance degradation for me (again, it may sound like an exaggeration, I'm not exaggerating).
> 
> By the way, thanks for your work on this amazing project!

Use IFEO to effectively disable it.

```
Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\compattelrunner.exe]
"Debugger"="systray.exe"

[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\wsqmcons.exe]
"Debugger"="systray.exe"
```

**Comment by Pcgeeks** [comment]
While holding down the Windows key and R, type regedit into the Run window. Input OK. In the next box, click Yes to access User Account Control.
Currently, navigate to HKEY LOCAL MACHINE > SOFTWARE > Policies > Microsoft > Windows > DataCollection. Here, choose New > DWORD by right-clicking on DataCollection (32-bit value). Double-click the new value after naming it "Allow Telemetry." Value data should now read 0 (zero), so click OK.
if you struggle to do it read a guide how to disable [CompatTelRunner.exe](https://geeksadvice.com/fix-microsoft-compatibility-telemetry-high-cpu-usage/)

**Comment by nslay** [comment]
I think have this registry item turned off already (I've followed that guide before IIRC). I'll have to check. No matter what I do to get rid of CompatTelRunner, it often gets reinstalled in future patch updates and then I find it running again.

**Comment by God-damnit-all** [comment]
> I think have this registry item turned off already (I've followed that guide before IIRC). I'll have to check. No matter what I do to get rid of CompatTelRunner, it often gets reinstalled in future patch updates and then I find it running again.

Yes, I also have telemetry disabled in this way, and I'm running Windows 10 Enterprise.
